ios/AppcuesFrameViewManager.swift

```diff
@@ -0,0 +1,122 @@
+import AppcuesKit
+
+@objc(AppcuesFrameViewManager)
+class AppcuesFrameViewManager: RCTViewManager {
+
+    override func view() -> WrapperView {
+        return WrapperView(uiManager: bridge.uiManager)
+    }
+
+    @objc override static func requiresMainQueueSetup() -> Bool {
+        return false
+    }
+}
+
+class WrapperView: UIView {
+    @objc var frameID: String? = nil
```

Comment:
Worth noting that this doesn't have a `didSet` to support changing the frameID, which I think it should?

Reply:
oh like the same native view can get recycled and re-used somehow?

Reply:
Yep, just thinking about our discussions lately around changing frame ids in the recycler case and then noticed the android implementation has a function that would re-register if the value changed. It's not a constant on the JS side, so to avoid any confusion I'll patch this up.

Reply:
Updated in c484207